### ISSUE TITLE
ERROR com.epam.reportportal.service.Launch - [18] ReportPortal execution error

### ISSUE THREAD

**Issue by amartanwar42** [reportportal/reportportal#694]
Hi Team,
I am getting below exception. Can somebody please help me.

> 16:56:16.342 [rp-io-0] ERROR com.epam.reportportal.service.Launch - [18] ReportPortal execution error
com.epam.reportportal.exception.InternalReportPortalClientException: Report portal is not functioning correctly. Response is not json
	at com.epam.reportportal.service.ReportPortalErrorHandler.handleError(ReportPortalErrorHandler.java:88)
	at com.epam.reportportal.service.ReportPortalErrorHandler.handle(ReportPortalErrorHandler.java:59)
	at com.epam.reportportal.restendpoint.http.HttpClientRestEndpoint$1.subscribe(HttpClientRestEndpoint.java:517)
	at io.reactivex.internal.operators.maybe.MaybeCreate.subscribeActual(MaybeCreate.java:45)
	at io.reactivex.Maybe.subscribe(Maybe.java:4096)
	at io.reactivex.internal.operators.maybe.MaybeCache.subscribeActual(MaybeCache.java:77)
	at io.reactivex.Maybe.subscribe(Maybe.java:4096)
	at io.reactivex.internal.operators.maybe.MaybeSubscribeOn$SubscribeTask.run(MaybeSubscribeOn.java:54)
	at io.reactivex.internal.schedulers.ScheduledDirectTask.call(ScheduledDirectTask.java:38)
	at io.reactivex.internal.schedulers.ScheduledDirectTask.call(ScheduledDirectTask.java:26)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)

**Comment by amartanwar42** [comment]
I have already verified below points:
properties are correct.
Report portal is up and running and i am able to login.

**Comment by amartanwar42** [comment]
I updated docker and it started working. Later i restarted my system and again i m getting same exception. Can some body please help me.

**Comment by amartanwar42** [comment]
Identified issue. 
ReportPortal was running on port localhost:8080 and jenkins was also running on 0.0.0.0:8080. 

I was able to access Jenkins  via localhost:8080 when report portal was not running. 
Since both can be accessed via localhost:8080 i was getting above exception (Assumption).

I stopped jenkins service and report portal started working.

ps -ef | grep jenkins
kill -9 id

Hope it helps others.
